easybuild/easyconfigs/r/resolos/resolos-0.3.5-GCCcore-11.3.0.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'resolos'
+version = '0.3.5'
+
+homepage = 'https://gitlab.unige.ch/resolos/resolos'
+description = """Resolos is a toolkit written in Python for maintaining reproducible environments 
+for scientific computations. It's main goal is to enable researchers to easily
+replicate environments through space (running code on HPC environment)
+and time (environment preservation for long term archival).
+For installation and detailed usage, check out the documentation."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
```

One last change, tests fails because this easyconfig also needs `binutils` to build. The `GCCcore` toolchain only provides the compiler and linker.
```suggestion

builddependencies = [('binutils', '2.38')]

```